src/platforms/esp32/main/platform_defaultatoms.c

```diff
@@ -74,6 +74,13 @@ static const char *const sta_got_ip_atom = "\xA" "sta_got_ip";
 static const char *const sta_connected_atom = "\xD" "sta_connected";
 static const char *const sta_disconnected_atom = "\x10" "sta_disconnected";
 static const char *const sta_dhcp_hostname_atom = "\xD" "dhcp_hostname";
+static const char *const ap_atom = "\x2" "ap";
+static const char *const ap_ssid_hidden_atom = "\xB" "ssid_hidden";
+static const char *const ap_max_connections_atom = "\xF" "max_connections";
```

Comment:
we should call constants with the closest name to the atom. So `max_connections_atom` would be better.

Reply:
Fixed